README.md

```diff
@@ -11,12 +11,12 @@ If something doesn’t work, please [file an issue](https://github.com/facebook/
 ## Quick Overview
 
 ```sh
-npx create-react-app my-app
+npm init create-react-app my-app
```

Comment:
It's `npm init react-app my-app`, per https://docs.npmjs.com/cli/init

Reply:
ahh yeah, fixing it now